server.py

```diff
@@ -43,7 +43,7 @@ def runGame(self, spriteGroup, wallList, pikaList, pikaBall, clickButton, txtImg
             sendList = ['0']*12   # send click list
             msg = ""   # send msg
             global NEWGAME, STARTDELAY
-            background = pygame.image.load('bg.jpg').convert()
+            background = pygame.image.load('.\\images\\bg.jpg').convert()
```

Comment:
images/bg.jpg?

Reply:
Will, because you put the background image in the "images" folder, you should let pygame load the image file in that place.

Reply:
Yes I know, I will change it to use os.path.join for OS compatibility.